e2e/steps_file.js

```diff
@@ -59,11 +59,10 @@ module.exports = function () {
       }
     },
 
-    seeCheckAnswers(checkAnswerTitle) {
-      this.click('Continue');
-      this.waitForElement('.check-your-answers');
-      this.see(checkAnswerTitle);
-      eventSummaryPage.submit('Save and continue');
+    async seeCheckAnswersAndCompleteEvent(button) {
```

Comment:
Is it not always going to be 'Save and continue'?

Reply:
no for the case submission controller it's 'Submit'